src/main/java/parking/beans/request/alterParking.java

```diff
@@ -0,0 +1,22 @@
+package parking.beans.request;
+
+public class alterParking {
```

Comment:
As far as I know we're using [CamelCase](https://www.wikiwand.com/en/CamelCase) for naming.